src/suite-design-guide/general-principles.rst

```diff
@@ -104,9 +104,27 @@ files they need are more robust.
 Installing Files At Start-up
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Use ``rose suite-run`` *file creation mode* or ``R1``
-install tasks to copy files to the self-contained suite run directory at
-start-up. Install tasks are preferred for time-consuming installations because
+Files can be installed on any remote platforms. As standard, Cylc installs the
+following directories: ``app``, ``bin``, ``etc``, ``lib``.
+Cylc supports adding custom directories and files to the file installation.
+
+E.g. dir1, dir2, file1, file2, you can add the following configuration to your
+:cylc:conf:`flow.cylc`, under the section
+:cylc:conf:flow.cylc[scheduler]install.
+To denote a directory, please add a trailing slash.  
+
+.. code-block:: cylc
+
+    [scheduler]
+        install = dir1/, dir2/, file1, file2
+
+.. note::
+
+   Please ensure files and directories to be installed are located in the top
```

Comment:
Some smart alec is bound ask why...

Also, I'm not a fan of the "pleases". Perhaps I'm just rude.